Makefile

```diff
@@ -1,6 +1,5 @@
-
-
-# Copyright (c) 2010 Chris Jones <jonesc@macports.org>
+# GNU Makefile for MacportsLegacySupport
+# Copyright (c) 2018 Christian Cornelssen
 #
```

Comment:
For all the Makefile stuff I defer to @cjones051073  -- might be nice to leave him credited, though, even though you  rewrote it, given the inspiration and collegiality of it all. What do you think?